fastapi/app/main.py

```
@@ -119,117 +56,154 @@ async def process_video(file: UploadFile = File(...)):
         }
 
-    except HTTPException as he:
-        raise he
+    except HTTPException:
+        raise
     except Exception as e:
-        logger.error(f"처리 실패: {str(e)}", exc_info=True)
-        raise HTTPException(500, "서버 내부 오류") from e
-
+        logger.error(str(e), exc_info=True)
+        raise HTTPException(500, "서버 오류")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)